### PR TITLE
Gen1: Don't localise number challenge instruction on initial render

### PR DESCRIPTION
**Pull request by denysoblohin-okta**
## Description:

Issue: On initial render of `NumberChallengeView` the argument for i18n key `oktaverify.numberchallenge.instruction` is null, so  calling `localize()` will dispatch 'okta-i18n-error'
https://github.com/okta/okta-signin-widget/blob/95ff4ac53b2517f0637509ece1cde70b74d68464/src/v1/views/mfa-verify/NumberChallengeView.js#L44
https://github.com/okta/okta-signin-widget/blob/95ff4ac53b2517f0637509ece1cde70b74d68464/src/v1/views/mfa-verify/NumberChallengeView.js#L20

https://developer.okta.com/docs/reference/api/authn/#response-example-waiting-for-3-number-verification-challenge-response

Test lcoally:
- add manually
```js
  document.addEventListener('okta-i18n-error', (ev) => {
    console.warn(JSON.stringify(ev.detail))
  });
```
- use `numberChallengeOv` mocks
- Notice warn in console (before clicking 'Send push')

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [x] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-700844](https://oktainc.atlassian.net/browse/OKTA-700844)

### Screenshot/Video:

https://github.com/okta/okta-signin-widget/assets/72614880/8c341248-4aa7-438d-b830-60735a393e86

### CODE REVIEW

File: src/v1/views/mfa-verify/NumberChallengeView.js [hunk 1]
```diff
@@ -15,9 +15,11 @@ import hbs from '@okta/handlebars-inline-precompile';
 export default View.extend({
   className: 'number-challenge-view',
   template: hbs`
-      <p data-se="number-challenge-instruction">
-        {{{i18n code="oktaverify.numberchallenge.instruction" bundle="login" arguments="number"}}}
-      </p>
+      {{#if number includeZero=true}}
```
Comment (denysoblohin-okta) [inline src/v1/views/mfa-verify/NumberChallengeView.js:18]: https://handlebarsjs.com/guide/builtin-helpers.html#includezero
Show instruction of number is 0

File: playground/mocks/data/api/v1/authn/mfa-number-challenge-ov.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      "profile": {
+        "credentialId": "exampleUser@example.com",
+        "deviceType": "SmartPhone_IPhone",
+        "keys": [
+          {
+            "kty": "PKIX",
+            "use": "sig",
+            "kid": "default",
+            "x5c": [
+              "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwd3kkRHieZUwY2wRaufKLuKWBHzy6oj8JjuXVyQJHzHJHyAjAo1jhey21v8WtxWMkb8soR1cg7iSf9kM/MjRxQJFKWNNhZDSgrMs/nc8RIO3xX1dWOVhNf51z/82S/+Wgo0ZRzrfM9iOFUwKDt5PoGe3d8rPsY3F5sJaw8lwAw9HqgI95RmRovta99S5zgh9DD3D57ckECKdCbe8HxFd+lkRLz1nl85FxEKLMaPa0vh8/AN8j14GSjoVogyLnF1468LEff7i2VL81HbUpO2PRQ7LEqTQWzmfcB0BULd499WydFIuwpV68c91VcGXWPUKHyXxZVB5SXSHgAgR45p8nQIDAQAB"
```
Comment (lesterchoi-okta) [inline playground/mocks/data/api/v1/authn/mfa-number-challenge-ov.json:30]: redact

**Review by lesterchoi-okta — COMMENTED** [review]
LGTM @shuowu-okta

**Review by shuowu-okta — APPROVED** [review]

**Review by mauriciocastillosilva-okta — COMMENTED** [review]
lgtm 🚢

**Pull request merged**